lib/application/presentation/detail_screen.dart

```diff
@@ -1,25 +1,294 @@
+import 'package:cached_network_image/cached_network_image.dart';
+import 'package:deep_conference/application/logic/speaker_state.dart';
 import 'package:deep_conference/domain/models/schedule_items.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+import '../../constants/my_colors.dart';
+import '../../constants/schedule_item_categories.dart';
+import '../../domain/models/speaker.dart';
+import '../logic/speaker_cubit.dart';
 
 class DetailScreen extends StatefulWidget {
```

Comment:
Ovaj Widget možda nebi bilo loše preimenovati u nešto tipa `ScheduleDetailsScreen`. Ovako je dosta nejasno na kakav Detail se misli :). Nemojte i fajl zaboraviti preimenovati onda.